_posts/en/newsletters/2020-09-16-newsletter.md

```diff
@@ -0,0 +1,216 @@
+---
+title: 'Bitcoin Optech Newsletter #115'
+permalink: /en/newsletters/2020/09/16/
+name: 2020-09-16-newsletter
+slug: 2020-09-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter recommends checking for nodes vulnerable to the
+InvDoS attack, briefly describes that attack, summarizes another attack
+against LN channels, and links to the announcement of the Crypto Open
+Patent Alliance.  Also included are our regular sections with releases,
+release candidates, and notable changes to popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+- **Check for nodes vulnerable to the InvDoS attack:** an attack
+  disclosed this week affects several old versions of full node software
+  (including altcoins).  According to the [paper][invdos paper],
+  affected Bitcoin software includes "Bitcoin Core v0.16.0, Bitcoin Core
+  v0.16.1, Bitcoin Knots v0.16.0, all beta versions of Bcoin up to
+  v1.0.0-pre, [and] all versions of Btcd up to v0.20.1-beta."  The
+  latest released versions of Bitcoin Core, Bitcoin Knots, Bcoin, and
+  Btcd all contain fixes.  See the *News* section below for details.
+
+## News
+
+- **Inventory out-of-memory Denial-of-Service attack (InvDoS):** Braydon
+  Fuller and Javed Khan [posted][invdos post] to the Bitcoin-Dev mailing
+  list an attack ([website][invdos site], [paper][invdos paper],
+  [CVE-2018-17145][]) they'd previously responsibly disclosed to the
```

Comment:
ISTM this was reported more than half a year earlier in https://github.com/bitcoin/bitcoin/issues/11822 and patched in https://github.com/bitcoin/bitcoin/pull/11824

Reply:
The reports in 11822 seem to only be about OOM during reindex---an event that should not be remotely triggerable and so isn't a vulnerability (it's just a bug).

Reply:
(Additionally, whether or not the problem was previously reported in Bitcoin Core, the paper is also about same fundamental problem in bcoin and btcd.)

Reply:
Agreed; sorry for the verification noise.